src/main/migrations/changesets/db_schema.xml

```diff
@@ -0,0 +1,158 @@
+    <changeSet author="scottfrazer" id="db_schema_other_table_alldb">
+        <createTable tableName="WORKFLOW_EXECUTION">
+            <column autoIncrement="true" name="WORKFLOW_EXECUTION_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_WORKFLOW_EXECUTION"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="WDL_URI" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="STATUS" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="START_DT" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="END_DT" type="datetime"/>
+        </createTable>
+
+        <createTable tableName="EXECUTION">
+            <column autoIncrement="true" name="EXECUTION_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_EXECUTION"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CALL_FQN" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="STATUS" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="JES_JOB">
+            <column autoIncrement="true" name="JES_JOB_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_JES_JOB"/>
+            </column>
+            <column name="EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="JES_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="JES_STATUS" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="LOCAL_JOB">
+            <column autoIncrement="true" name="LOCAL_JOB_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_LOCAL_JOB"/>
+            </column>
+            <column name="EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="PID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="COMMAND" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="RC" type="INT"/>
+        </createTable>
+
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB"/>
+        </modifySql>
```

Comment:
I'm pretty sure ALL tables should be InnoDB.  As far as I can tell from googling around, MyISAM doesn't support foreign key constraints or ACID transactions.